### ISSUE TITLE
problem with demo

### ISSUE THREAD

**Issue by sunshineying** [rrag/react-stockcharts#100]
Hi. When I use 

```javascript
var emaConfig = ema()
	.windowSize(20) 
	.source(d => d.close)
	.skipUndefined(true) 
	.merge((d, c) => {d.emaConfig = c}) 
	.accessor(d => d.emaConfig)
	.stroke("blue") 
```
It's normal. But when I use 
```javascript
var emaConfig  = ema()
	.windowSize(child.emaArgument.period) 
	.source(d => d[`${child.emaArgument.source}`])
	.merge((d, c) => {d.emaConfig = c}) 
	.accessor(d => d.emaConfig)
	.stroke(child.emaArgument.stroke); 
```
and child.emaArgument is an object, it can be changed by a component. 
The LineSeries and CurrentCoordinate did not display.
Then I **click the EventCapture area**, the LineSeries and CurrentCoordinate show up.
When I change the child.emaArgument.stroke, the line's color changes immediately.
When I change the child.emaArgument.souce, the line changes after I click the EventCapture area.
When I change the child.emaArgument.period, the line did not change, and even disappeared.
I don't know what's wrong with it.

![image](https://cloud.githubusercontent.com/assets/18010449/16729858/bcde47ba-47a1-11e6-8a1e-5bea51f18961.png)

![image](https://cloud.githubusercontent.com/assets/18010449/16729817/8748c4cc-47a1-11e6-95c6-cae003a9bc97.png)

**Comment by rrag** [comment]
Without a plunker I cannot offer much help. I will only be guessing.

Is your problem that the initial render works but when you change the values of `windowSIze` or `source`, they are not reflected immediately and it takes a click to see them new values?

**Comment by sunshineying** [comment]
sorry, I can't provide a plunker now.  I provide part of my code,  maybe you can give me some advise. thanks. 
http://plnkr.co/edit/02duEweznWmaFae2tqfI?p=preview
![image](https://cloud.githubusercontent.com/assets/18010449/16735004/0bd91b0c-47ba-11e6-91b4-a06512c5070a.png)
![image](https://cloud.githubusercontent.com/assets/18010449/16735288/1d888972-47bb-11e6-87e4-bf1390f043e6.png)

**Comment by sunshineying** [comment]
Not really. I am  trying to find another way to figure it out.  I'd like to know when next version  can come out.

**Comment by rrag** [comment]
almost there

http://rrag.github.io/react-stockcharts-next/documentation.html#/coming_soon

and then update documentation and change log

**Comment by sunshineying** [comment]
Hi. I install the latest package by `npm install react-stockcharts@next` , the version is 0.5.0-alpha1.
But I find that it lacks of some components, such as `discontinuousTimeScaleProvider`, and the `financeEODDiscontinuousScale` is not be moved.
Is it the 0.5 version not finished?

**Comment by rrag** [comment]
0.5 is not published yet. the alpha version was published before a lot of changes were made.

**Comment by rrag** [comment]
published 0.5 now
